ci/d2-convert-tag.sh

```diff
@@ -29,5 +24,5 @@ then
 	set +x
 	git -c "credential.https://github.com.username=${OCEAN_D2_USER}" \
 		-c "core.askPass=./ci/askpass.sh" \
-		push "https://github.com/sociomantic-tsunami/ocean-d2.git" "$d2tag"
+		push "https://github.com/sociomantic-tsunami/ocean.git" "$d2tag"
```

Comment:
`d2tag` is set to `$TRAVIS_TAG+d2.auto` here https://github.com/sociomantic-tsunami/ocean/pull/192/files/d484b914bb329fcc645597a2ceff953a13b78d47#diff-c4405eca91b5ba6f9ea44c38cdd19ebfR20